travel_platform/travel_app/views.py

```diff
@@ -3,10 +3,28 @@
 
 from django.http import Http404
 
+from django.http import JsonResponse
+
 from django.shortcuts import render
 
 from .models import Query
 
 
 def index(request):
     return render(request, 'travel_app/main.html')
+
+
+def coord(request):
+    """
+    View added for checking of interaction of map from landing page with 
+    server.
+    :param request: 
+    :return: 
+    """
+    if 'lon' in request.GET:
```

Comment:
I guess, it's an amazing place for TODO or FIXME, because it doesn't make any sense without it.

Also, I suggest to replace these few lines of code, because it may work fine if `lon` exists, but it's not clear if it's expected to have missing `lat` here.
So, I suggest to use simple `request.GET.get('lon')` and `request.GET.get('lat')` instead.